.github/workflows/test.yml

```diff
@@ -90,6 +90,7 @@ jobs:
         testcase:
           - TestE2ESimple
           - TestStateSync
+          - TestStateMocha
```

Comment:
we probably don't need to run this on every PR 🤔 , leaving it here now so it runs in CI.